### ISSUE TITLE
Missing "claims_supported" part of openid-configuration

### ISSUE THREAD

**Issue by stevenandres** [juanifioren/django-oidc-provider#332]
Forgive the perhaps obvious question -- trying to interface a working OIDC provider setup with a third-party system that is complaining that the "claims_supported" dictionary is not provided at .well-known/openid-configuration

How do we enable that section? I'm a bit lost in the weeds of the documentation. I don't want to create any custom types, just the standard ones. I was told by this third party that our openid-configuration should look similar to this demo response:

https://abc123-admin.oktapreview.com/.well-known/openid-configuration

**Comment by geoff-va** [comment]
https://github.com/juanifioren/django-oidc-provider/blob/master/oidc_provider/views.py#L260-L292

Doesn't look like there are currently any mechanisms to support this. It's not part of the current built in response, so would need to be added.

**Comment by stevenandres** [comment]
Correct -- it is not required but I have run into a couple of large third party enterprise systems that are treating it as if it were required (not adhering to standards). In any case, it might be a "nice-to-have" for future revisions of the library. Thanks for the reply!

**Comment by geoff-va** [comment]
You bet - does sound like it would be a useful feature to have. Might be worth re-opening this issue or create a new request as an enhancement if you're interested!

**Comment by stevenandres** [comment]
Can I just use an **issue** as an enhancement? Didn't know what was polite way to do it. I'll reopen and see what we get.

**Comment by geoff-va** [comment]
yup! You may want to rename it so it's more indicative of the request - something more like: `Add claims_supported to open-id configuration` or whatever you see fit, and then reference pertinent specs in your request.

At some point a maintainer can tag the issue as an Enhancement (See labels tab in the Issues section).
